### PR TITLE
Update strategies-migrating-in-production.md

### PR DESCRIPTION
**Pull request by KSchlobohm**
## Summary

Modified wording to make a stronger recommendation for multi-targeting class libraries when applying the strangler pattern

### CODE REVIEW

File: docs/architecture/porting-existing-aspnet-apps/strategies-migrating-in-production.md [hunk 1]
```diff
@@ -43,9 +43,13 @@ Eventually, the entire facade layer corresponds to the new, modern implementatio
...
+Multi-targeting is recommended for large apps that will be migrated over time and for teams applying the Strangler pattern approach. This approach can address BindingRedirect and package restoration challenges that surface from mixing [PackageReference](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files) and [packages.config](https://docs.microsoft.com/en-us/nuget/reference/packages-config) restore styles. There are two options available for code that must run in both .NET Framework and .NET Core environments.
 
-These techniques allow a single common codebase to be maintained while new functionality is added and (parts of) the app are ported to use .NET Core.
+* [preprocessor `#if`](../../csharp/language-reference/preprocessor-directives.md#conditional-compilation) directives allow you to implement different functionality or use different dependencies when run in .NET Framework versus .NET Core.
```
Comment (KSchlobohm) [inline docs/architecture/porting-existing-aspnet-apps/strategies-migrating-in-production.md:48]: is there a way to make this link language agnostic?

File: docs/architecture/porting-existing-aspnet-apps/strategies-migrating-in-production.md [hunk 2]
```diff
@@ -43,9 +43,13 @@ Eventually, the entire facade layer corresponds to the new, modern implementatio
...
+Multi-targeting is recommended for large apps that will be migrated over time and for teams applying the Strangler pattern approach. This approach can address BindingRedirect and package restoration challenges that surface from mixing [PackageReference](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files) and [packages.config](https://docs.microsoft.com/en-us/nuget/reference/packages-config) restore styles. There are two options available for code that must run in both .NET Framework and .NET Core environments.
```
Comment (KSchlobohm) [inline docs/architecture/porting-existing-aspnet-apps/strategies-migrating-in-production.md:46]: Wasn't sure what the relative URL path would be

**Comment by nishanil** [comment]
@IEvangelist Can you help fix the link-related warnings, please?

**Comment by IEvangelist** [comment]
Hi @nishanil - all good now.

**Review by nishanil — APPROVED** [review]
Looks good to me!

**Pull request merged**